pkg/pidfile/pidfile.go

```diff
@@ -0,0 +1,44 @@
+package pidfile
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"strconv"
+)
+
+type PidFile struct {
```

Comment:
reminding again: consider making this `type PidFile string` as it doesn't seem like it'll get any other fields.